functions/src/graphql/resolvers/collections/menuItems.resolver.ts

```diff
@@ -22,6 +22,16 @@ export class MenuItemsResolver extends withBoilerplate("MenuItemsResolver") {
     return this.menuItemLoader.getMenuItem(id);
   }
 
+  @Query(() => MenuItemCollection, {
```

Comment:
This wont ever be needed. Also what the id here? How can I pass one menuItem id and get a collection?